Client/OrganizationsApi.cs

```diff
@@ -139,9 +139,9 @@ public async Task<Organization> FindOrganizationByIdAsync(string orgId)
         /// List all organizations.
         /// </summary>
         /// <returns>List all organizations</returns>
-        public async Task<List<Organization>> FindOrganizationsAsync()
+        public Task<List<Organization>> FindOrganizationsAsync()
         {
-            return await _service.GetOrgsAsync().ContinueWith(t => t.Result.Orgs);
+            return _service.GetOrgsAsync().ContinueWith(t => t.Result.Orgs);
```

Comment:
Why are `async` and `await` removed?

```csharp
public async Task<List<Organization>> FindOrganizationsAsync()
{
    return await _service.GetOrgsAsync().ContinueWith(t => t.Result.Orgs);
}
```

Reply:
I thought you asked for it
![image](https://user-images.githubusercontent.com/3382669/87252310-95699400-c47a-11ea-98ca-8c5103663fbc.png)

Reply:
Sorry for late. We've fixed the useless `ContinueWith` in #108. Could you rebase sources with master branch?

Reply:
I don't sure how to rebase it, i already push it to the fork.
I can merge and remove this issue